### PR TITLE
Added proposal table

### PR DESCRIPTION
**Pull request by jasonwilliams**
Follow up PR to devEnvironment, this creates a new table (hardcoded for now) for discussion

tctable

### CODE REVIEW

**Comment by ljharb** [comment]
How can we ensure that the proposals repo remains the source of truth? In other words, how can we make sure the website is never out of sync with the repo?

**Comment by littledan** [comment]
@ljharb We'll have to manually maintain data here, in order to give more explanations of features than we have in the proposals repository, but also more concise than you can find from current READMEs.

**Comment by ljharb** [comment]
I wonder if we could have a build process in this repo that only ends up rendering proposals’ extra info when the proposal is in the main repo, and that renders a placeholder section for proposals in the main repo that lack extra info here?

**Comment by codehag** [comment]
@ljharb We were thinking about making this dynamic as well, once we have a better idea of what is needed. For the moment we are just seeing what this looks like as a proof of concept and help us figure out exactly what information we need, and where best to store that information

**Review by codehag — COMMENTED** [review]
I like the direction of this a lot 👍

File: README.md [hunk 1]
```diff
@@ -1,8 +1,14 @@
-
 # tc39 web draft
 
 
 This is a sketch of a potential website to address an information orientation need for the tc39, see [website](http://tc39.github.io/tc39-web-draft/).
 
 To learn more please get in touch with yulia at mozilla.com
 
+## Setting up your environment
```
[the remark this answers is not in the document]
Comment (jasonwilliams) [reply README.md:8]: Yeah, i had to branch of the previous PR so i could work locally, if we can merge that one hopefully we can lose those changes from this PR

File: .editorConfig [hunk 2]
```diff
@@ -14,15 +14,15 @@ indent_style = space
 indent_size = 2
 
 [**.css]
-indent_style = tab
+indent_style = space
```
Comment (ljharb) [inline .editorConfig]: aww :-(

File: _site/assets/img/menu-down.svg [hunk 3]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright © 2015 Apple Inc. All rights reserved. -->
```
Comment (xtuc) [inline _site/assets/img/menu-down.svg:2]: Depending on the licences we should have a dedicated document with all the licences we are using.

Comment (jasonwilliams) [reply _site/assets/img/menu-down.svg:2]: Good point, i reckon i could replace that chevron with a creative commons one

Comment (zoepage) [reply _site/assets/img/menu-down.svg:2]: Maybe this could be helpful: https://feathericons.com/

**Comment by xtuc** [comment]
I have no bandwidth due to a few confs. Should be better next week.

**Comment by codehag** [comment]
Closing this, as it has been rebased and was necessary for @Regaddi 's pr -- @xtuc feel free to branch off of master. if there are any issues with conflicts let me know

**Pull request closed without merging**